modules/apps/segments/segments-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -89,10 +91,20 @@ request.setAttribute("view.jsp-eventName", eventName);
 							/>
 						</c:when>
 						<c:otherwise>
-							<liferay-ui:icon
-								message="source.dxp"
-								src='<%= PortalUtil.getPathContext(request) + "/assets/dxp-icon.svg" %>'
-							/>
+							<c:choose>
+								<c:when test='<%= releaseInfo.contains("CE") %>'>
+									<liferay-ui:icon
+										message="source.ce"
+										src='<%= PortalUtil.getPathContext(request) + "/assets/ce-icon.svg" %>'
```

Comment:
we should try never to use Util classes in JSPs.

Reply:
We could add a method in SegmentsDisplayContext to provide the proper icon (message and src)